ext/websocket/lib.rs

```diff
@@ -483,7 +483,9 @@ pub async fn op_ws_next_event(
     Some(Ok(Message::Pong(_))) => NextEventResponse::Pong,
     Some(Err(e)) => NextEventResponse::Error(e.to_string()),
     None => {
-      state.borrow_mut().resource_table.close(rid).unwrap();
+      // No next event was found, presumably the stream closed while we were waiting.
+      // Ignore the error and return Closed state.
+      let _ = state.borrow_mut().resource_table.close(rid);
```

Comment:
We should move this out to a new PR. It might be a fix for #15726, but I'm not sure because I don't know anything about websockets.

Reply:
Also, note I disabled this flaky test in https://github.com/denoland/deno/pull/15997

Reply:
Removed the commits.

I have no idea if this is a correct fix to the flaky test as I likewise know nothing about websockets :D